<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startBut">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>263</y>
      <width>150</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_pred">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add predator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plant">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add plant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_prey">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add prey</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="count_pred">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="count_prey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="count_plant">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="creature_list">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>280</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>20</width>
      <height>400</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>20</width>
      <height>400</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>395</y>
      <width>700</width>
      <height>10</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="simBut">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>450</y>
      <width>150</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>SIMULATE</string>
    </property>
   </widget>
   <zorder>creature_list</zorder>
   <zorder>startBut</zorder>
   <zorder>button_pred</zorder>
   <zorder>button_plant</zorder>
   <zorder>button_prey</zorder>
   <zorder>count_pred</zorder>
   <zorder>count_prey</zorder>
   <zorder>count_plant</zorder>
   <zorder>line_1</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>simBut</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
